tests/favicon.svg

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Ahogy a css-t copyzod a tesztbe, úgy nem lehetne ezt is? Vagy ha gitbe berakunk egy teszt favicon svgt akkor az lehetne egy stub, nem kéne a teljes tartalom.

Reply:
Én csak a meglévő mintát követtem ahogy az `.ico`-nál is csináltad.

Reply:
Kész, átmásolom `Makefile`-ból.